### PR TITLE
5th Milestone of Calculator Project, React

### PR DESCRIPTION
**Pull request by AnthonyTC89**
Hello! This Milestone is about the interactions of the logic and the buttons of the App.
I follow all the [Requirements](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896095/) of the Project.
All the buttons are working correctly. I did many tests but if there is something wrong let me know.
This is the link of the Live Demo is [here](https://feature-interactions.d1mt0f332rdrd4.amplifyapp.com/)
The App will be deployed on Heroku after the approval.

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -3,13 +3,30 @@ import Display from './Display';
...
-    <Display />
-    <ButtonPanel />
-  </div>
-);
+const App = () => {
+  const [state, setState] = React.useState({
+    total: null,
+    next: null,
+    operation: null,
+  });
```
Comment (jofer86) [inline src/components/App.js]: **[ REQUIRED ]** As the scope of the project requires, we need to make our App component stateful, I can see you used the useState hook for this, this while it gets the job done, it is not within the scope of this project, as you can see [here](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896095/) 
image

Let us make sure our App is a stateful component and you are good to go.

**Review by jofer86 — CHANGES_REQUESTED** [review]
## 👋 👋 Hello, @AnthonyTC89 I will be conducting the review for this milestone.

- As you can see from the comment below, there is only one change that is required of you in order to approve this PR, please tend to it and we shall be good to go!

----------------------
Cheers,
Jorge Fernando Rincón Figueroa
Technical Support Engineer
**@jofer86 in Slack**

File: src/components/App.js (hunk 1 above)
Comment (AnthonyTC89) [reply src/components/App.js]: the changes were done.

**Review by bolah2009 — APPROVED** [review]
It's a 👍🏾 from me!

### :heavy_check_mark: Approved!

Good job implementing the suggested changes.

#### Please update your README with description. of the project and live screen

Cheers
[Bola](https://bolabuari.com/)

**Pull request merged**